source_gen/lib/source_gen.dart

```diff
@@ -10,6 +10,6 @@ export 'src/generator.dart'
     show Generator, InvalidGenerationSource, InvalidGenerationSourceError;
 export 'src/generator_for_annotation.dart' show GeneratorForAnnotation;
 export 'src/library.dart' show AnnotatedElement, LibraryReader;
-export 'src/span_for_element.dart' show spanForElement;
+export 'src/span_for_element.dart' show spanForElement, spanForElement2;
```

Comment:
Need a changelog update

Reply:
It will never be published, we will restore the name with different types.